Mathlib/Algebra/Group/Pointwise/Set/Card.lean

```diff
@@ -47,6 +48,12 @@ lemma _root_.Cardinal.mk_inv (s : Set G) : #↥(s⁻¹) = #s := by
 lemma natCard_inv (s : Set G) : Nat.card ↥(s⁻¹) = Nat.card s := by
   rw [← image_inv, Nat.card_image_of_injective inv_injective]
 
+@[to_additive (attr := simp)]
+lemma encard_inv (s : Set G) : s⁻¹.encard = s.encard := by simp [encard, PartENat.card]
+
+@[to_additive (attr := simp)]
+lemma ncard_inv (s : Set G) : s⁻¹.ncard = s.ncard := by simp [ncard]
+
 @[to_additive (attr := deprecated (since := "2024-09-30"))] alias card_inv := natCard_inv
```

Comment:
```suggestion
@[to_additive (attr := deprecated (since := "2024-09-30"))] alias card_inv := natCard_inv

@[to_additive (attr := simp)]
lemma encard_inv (s : Set G) : s⁻¹.encard = s.encard := by simp [encard, PartENat.card]

@[to_additive (attr := simp)]
lemma ncard_inv (s : Set G) : s⁻¹.ncard = s.ncard := by simp [ncard]
```
to keep the alias closer

Reply:
I was purposefully flushing the aliases to the bottom because they are mostly just noise. What's your rationale for wanting aliases closer to the declaration they already reference?

Reply:
Since they reference the declaration, it's easier to identify which lemma is being referenced rather than needing to scroll to find it. In addition, in the docs, it's useful to have these lemmas next to each other rather than having other things in between.